src/screens/Assets/AssetsList.js

```diff
@@ -243,27 +290,56 @@ class AssetsList extends React.Component<Props, State> {
           if (scrollViewRef) scrollViewRef.setNativeProps({ scrollEnabled: shouldAllowScroll });
         }}
       >
-        <ListItemWithImage
-          onPress={() => {
-            navigation.navigate(ASSET,
-              {
-                assetData: {
-                  ...props,
-                  tokenType: TOKENS,
+        {symbol !== BTC &&
```

Comment:
So much repetition :)
Maybe we could check if symbol === BTC when setting onPress value (as this the only value that is different)? 
CheckAuth can be rendered next to FlatList in render method :)

Reply:
Yup this can be trimmed down, but for CheckAuth it will be rendered at this position per BTC only. Otherwise it would be a bigger refactor can be done in a chore when multiple assets need activation like this one